service/src/main/java/bio/terra/workspace/app/configuration/external/PolicyServiceConfiguration.java

```diff
@@ -8,11 +8,13 @@
 import java.io.IOException;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.context.annotation.Configuration;
 
 /** Configuration for managing connection to Terra Policy Service. * */
 @Configuration
 @EnableConfigurationProperties
+@EnableCaching
```

Comment:
I am mistrustful of Spring caching. We tried to use it in TDR and it did not work well. We hand-coded instead. It looks like you have not configured a cache provider.

See [here](https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-caching.html#boot-features-caching-provider) for choices and how to configure. Also [here](https://docs.spring.io/spring-framework/docs/5.1.8.RELEASE/spring-framework-reference/integration.html#cache) for more info.

Reply:
I'm using spring-boot-starter-cache so I don't need to register a cache provider.

"When using Spring Boot, the mere presence of the starter package on the classpath alongside the EnableCaching annotation would register the same ConcurrentMapCacheManager. So there is no need for a separate bean declaration."

[spring-boot-starter-cache](https://search.maven.org/search?q=g:org.springframework.boot%20a:spring-boot-starter-cache)

You suggested caching in another ticket hence I implemented it. Fine to not cache :)